doc/howtos/rdtraining/15_qwebintro.rst

```diff
@@ -0,0 +1,133 @@
+
+    .. image:: 15_qwebintro/media/kanban.png
+        :align: center
+        :alt: Kanban view
+
+In our estate application, we would like to add a Kanban view to display our properties. Kanban
+views are standard Odoo views (such as the form or list view), but their structure is much more
+flexible. In fact, the structure of each card is a mix of form elements (including basic HTML)
+and QWeb. The definition of a Kanban view is similar to the definition of the list and form
+views, except that their root element is ``<kanban>``. In its simplest form, a Kanban view
+looks like:
+
+.. code-block:: xml
+
+    <kanban>
+        <templates>
+            <t t-name="kanban-box">
+                <div class="oe_kanban_global_click">
+                    <field name="name"/>
+                </div>
+            </t>
+        </templates>
+    </kanban>
+
+Let's break down this example:
+
+- ``<templates>``: defines a list of :ref:`reference/qweb` templates. Kanban views *must* define at
+  least one root template ``kanban-box``, which will be rendered once for each record.
+- ``<t t-name="kanban-box">``: ``<t>`` is a placeholder element for QWeb directives. In this case,
+  it is used to set the ``name`` of the template to ``kanban-box``
+- ``<div class="oe_kanban_global_click">``: the ``oe_kanban_global_click`` makes the ``<div>``
+  clickable to open the record.
+- ``<field name="name"/>``: this will add the ``name`` field to the view.
+
+.. exercise:: Minimal kanban view
+
+    Using the simple example provided, create a minimal Kanban view for the properties. The
+    only field to display is the ``name``.
+
+    Tip: you must add ``kanban`` in the ``view_mode`` of the corresponding
+    ``ir.actions.act_window``.
+
+Once the Kanban view is working, we can start improving it. If we want to display an element
+conditionally, we can use the ``t-if`` directive (see :ref:`reference/qweb/conditionals`).
+
+.. code-block:: xml
+
+    <kanban>
+        <field name="state"/>
+        <templates>
+            <t t-name="kanban-box">
+                <div class="oe_kanban_global_click">
+                    <field name="name"/>
+                </div>
+                <div t-if="record.state.raw_value == 'new'">
+                    This is new!
+                </div>
+            </t>
+        </templates>
+    </kanban>
+
+We added a few things:
+
+- ``t-if``: the ``<div>`` element is rendered if the condition is true.
+- ``record``: an object with all the requested fields as its attributes. Each field has
+  two attributes ``value`` and ``raw_value``, the former is formatted according to current
+  user parameters, the latter is the direct value from a :meth:`~odoo.models.Model.read`.
+
+In the above example, the field ``name`` was added in the ``<templates>`` element, but ``state``
+is outside. When we need the value of a field but not display it in the view, it is possible to
```

Comment:
Maybe clarify / expand on this e.g. "the kanban view fetches as little as it can so the value isn't even on the pseudo-record unless we request it"?